### PR TITLE
fixes for PR-194

### PR DESCRIPTION
**Pull request by tiadobatima1**

### CODE REVIEW

File: community/faas/docs/testing.md [hunk 1]
```diff
@@ -72,13 +72,13 @@ are often created, and deleted via *teardown()* function.
 Due to the fact that a DM config file needs to be located relative to the
 templates it uses, these configs are usually created in the root of the
 project, for example, in the network template mentioned, the config
-`.${PROJECT_NAME}-network.yaml` will be temporarily created (and deleted at the
-end of the execution).
+`.${FAAS_PROJECT_NAME}-network.yaml` will be temporarily created (and deleted
```
Comment (sourced-vince) [inline community/faas/docs/testing.md]: FASS_PROJECT_ID?

Comment (tiadobatima1) [reply community/faas/docs/testing.md]: I think it's `FAAS`... that's what I've been using everywhere

Comment (tiadobatima1) [reply community/faas/docs/testing.md]: Ohh... gotcha.

Comment (tiadobatima1) [reply community/faas/docs/testing.md]: fixed

**Pull request merged**